aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/Orchestrator.java

```diff
@@ -97,6 +103,11 @@ public Orchestrator(
         Merger merger = new Merger(mutationOutbox, versionRepository, localStorageAdapter);
         SyncTimeRegistry syncTimeRegistry = new SyncTimeRegistry(localStorageAdapter);
 
+        this.localStorageAdapter = localStorageAdapter;
+        syncableModels = Observable.fromIterable(modelProvider.models())
+            .toList()
+            .blockingGet();
```

Comment:
This seems to be a frivolous construct. Isn't this just equivalent to using `modelProvider.models()` directly?

Reply:
Yeah...initially, I was going to filter based on isSyncable, but we don't have such a flag for Android.